.pre-commit-config.yaml

```diff
@@ -44,3 +44,7 @@ repos:
       args: ["--remove"]
     - id: mixed-line-ending
       args: ["--fix=lf"]
+- repo: https://gitlab.com/daverona/pre-commit/cpp
```

Comment:
This repo seems not to be maintained anymore. What about this hook at GH https://github.com/pre-commit/mirrors-clang-format